python_modules/libraries/dagster-components/dagster_components/components/sling_replication_collection/component.py

```diff
@@ -32,6 +36,22 @@
 SlingMetadataAddons: TypeAlias = Literal["column_metadata", "row_count"]
 
 
+class DagsterSlingTranslatorAutomationCondition(DagsterSlingTranslator):
```

Comment:
There is prob a better way to do this that involves a more careful rewrite of the wrapped translator logic, this is a stopgap really

Reply:
`DagsterSlingTranslatorOverrideAutomationCondition`